### ISSUE TITLE
1.12.2 - Unhandled exception executing 'anvil' in com.sk89q.bukkit.util.DynamicPluginCommand(anvil); java.lang.NoSuchFieldError: c

### ISSUE THREAD

**Issue by mibby** [Siguza/VirtualPack#45]
VirtualPack dev 2.8.13
Paper dev 1209 (Spigot 1.12.2)

@Siguza Broke in MC / spigot 1.12.2 update.

```
[03:50:51] [Server thread/INFO]: player issued server command: /anvil
[03:50:51] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing 'anvil' in com.sk89q.bukkit.util.DynamicPluginCommand(anvil)
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:160) ~[patched_1.12.2.jar:git-Paper-1209]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:676) ~[patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1431) ~[patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1236) ~[patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:5) ~[patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:14) ~[patched_1.12.2.jar:git-Paper-1209]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_144]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_144]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:843) [patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:424) [patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:767) [patched_1.12.2.jar:git-Paper-1209]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:665) [patched_1.12.2.jar:git-Paper-1209]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_144]
Caused by: java.lang.NoSuchFieldError: c
	at net.drgnome.virtualpack.components.VAnvil.<init>(VAnvil.java:27) ~[?:?]
	at net.drgnome.virtualpack.VPack.openAnvil(VPack.java:1295) ~[?:?]
	at net.drgnome.virtualpack.VCommands.anvil(VCommands.java:1699) ~[?:?]
	at net.drgnome.virtualpack.VCommands.tools(VCommands.java:183) ~[?:?]
	at net.drgnome.virtualpack.VCommands.cmd(VCommands.java:120) ~[?:?]
	at net.drgnome.virtualpack.VCommands.onCommand(VCommands.java:77) ~[?:?]
	at com.sk89q.bukkit.util.DynamicPluginCommand.execute(DynamicPluginCommand.java:54) ~[?:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:151) ~[patched_1.12.2.jar:git-Paper-1209]
	... 14 more
```

**Comment by mibby** [comment]
For reference to update to 1.12.2;

```
                                <replacement>
                                    <token>#FIELD_CONTAINER_2#</token> <!-- [List] -->
                                    <value>b</value>
                                </replacement>
                                <replacement>
                                    <token>#FIELD_CONTAINER_3#</token> <!-- [List] -->
                                    <value>c</value>
                                </replacement>
```

B changed to ```items```, C changed to ```slots```.
https://hub.spigotmc.org/stash/projects/SPIGOT/repos/craftbukkit/commits/9a1f5ee80a8665b0c388758003e2bda1ee924084#src/main/java/org/bukkit/craftbukkit/inventory/CraftContainer.java

**Comment by lululombard** [comment]
I can confirm this issue.

**Comment by Siguza** [comment]
Should be fixed with 28062b0c101ef1d51e58ed1734e0121bdd341d73. Thanks for digging through the changes, @mibby. :)
